.github/pull_request_template.md

```diff
@@ -13,21 +13,14 @@ the terms of the Apache 2.0 license.
 
 ## PR Checklist
 
-`[Author TODO: Meet these criteria. Where there are two options, keep one.]`
+`[Author TODO: Meet these criteria.]`
 `[Reviewer TODO: Verify that these criteria are met. Request changes if not]`
 
+- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).
 - [ ] All commits in this PR are signed (`git commit -s`).
-- [ ] Either this PR is linked to an issue, or, the reason for this PR is
-      clearly provided.
+- [ ] The reason for this PR is clearly provided (issue no. or explanation).
 - [ ] The description of changes is clear and encompassing.
-- [ ] Either no docs need to be updated as part of this PR, or, the required
-      doc changes are included in this PR. Docs in scope are all `*.md` files
-      located either in the repository root, or in the `docs/` directory.
-- [ ] Either no code has been touched, or, code-level documentation for touched
-      code is included in this PR.
-- [ ] Either no API changes are included in this PR, or, the API changes are
-      reflected in `firecracker/swagger.yaml`.
-- [ ] Either the changes in this PR have no user impact, or, the changes in
-      this PR have user impact and have been added to the `CHANGELOG.md` file.
-- [ ] Either no new `unsafe` code has been added, or, the newly added `unsafe`
-      code is unavoidable and properly documented.
+- [ ] Any required documentation changes (code and docs) are included in this PR.
+- [ ] Any newly added `unsafe` code is properly documented.
+- [ ] Any API changes are reflected in `firecracker/swagger.yaml`.
+- [ ] The `CHANGELOG.md` file is up to date.
```

Comment:
Maybe leave a few words to indicate only user-facing changes should go in the CHANGELOG.

Reply:
Like:
Any user-facing changes contained in this PR are reflected in CHANGELOG.md

Also I would put Any API changes *contained in this PR* are reflected in `firecracker/swagger.yaml`.

Reply:
Reworded.